jose_alba.md

```diff
@@ -0,0 +1,5 @@
+Three to five ideas about frontend:
+1. MySpace anyone?
```

Comment:
lolll totally what i think of every time "coding" comes to mind

Reply:
MYSPACE!!! where it all started